pom.xml

```diff
@@ -20,10 +20,72 @@
     </license>
   </licenses>
 
+  <profiles>
+    <profile>
+      <id>internal-release</id>
+      <activation>
+        <property>
+          <name>useInternalRepo</name>
+          <value>true</value>
+        </property>
+      </activation>
+      <properties>
+      </properties>
+      <distributionManagement>
+        <repository>
+          <id>clx-releases</id>
+          <url>${env.REPO_URL}/repository/clx-releases-hosted/</url>
+        </repository>
+        <snapshotRepository>
+          <id>clx-snapshots</id>
+          <url>${env.REPO_URL}/repository/clx-snapshots-hosted/</url>
+        </snapshotRepository>
+      </distributionManagement>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.sonatype.plugins</groupId>
+            <artifactId>nexus-staging-maven-plugin</artifactId>
+            <version>1.6.13</version>
+            <extensions>true</extensions>
+            <configuration>
+              <serverId>clx-snapshots</serverId>
+              <nexusUrl>${env.REPO_URL}</nexusUrl>
+              <skipStaging>true</skipStaging>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <profile>
+      <id>prod-repository</id>
```

Comment:
internal release => public release ?

Reply:
what do you mean?

Reply:
It is minor thing, but I've meant profiles names. Internal-release deploys to internal repo and the second one currently is  - prod-repository